wiki/redirect.yaml

```diff
@@ -1,636 +1,1174 @@
+"sb_load":                     "Beatmap_Editor/SB_Load"
+"song_setup":                  "Beatmap_Editor/Song_Setup"
+"timelines":                   "Beatmap_Editor/Timelines"
+"timeline":                    "Beatmap_Editor/Timelines"
+"timing":                      "Beatmap_Editor/Timing"
+
+"beatmapping":                 "Beatmapping"
+
+"beatmaps":                    "Beatmaps"
+"beatmap":                     "Beatmaps"
+"map":                         "Beatmaps"
+"maps":                        "Beatmaps"
+"beatmap_pack":                "Beatmaps/Packs"
+"beatmap_packs":               "Beatmaps/Packs"
+"beatmappack":                 "Beatmaps/Packs"
+"beatmappacks":                "Beatmaps/Packs"
+"map_pack":                    "Beatmaps/Packs"
+"map_packs":                   "Beatmaps/Packs"
+"mappack":                     "Beatmaps/Packs"
+"mappacks":                    "Beatmaps/Packs"
+"pack":                        "Beatmaps/Packs"
+"packs":                       "Beatmaps/Packs"
+
+"beginner":                    "Beginner"
+
+"best_of":                     "Best_Of"
+"best_of_2012":                "Best_Of/2012"
+
+"chat_console":                "Chat_Console"
+
+"difficulties":                "Difficulties"
+"difficulty":                  "Difficulties"
+
+"easy":                        "Difficulties/osu!/Easy"
+"expert":                      "Difficulties/osu!/Expert"
+"hard":                        "Difficulties/osu!/Hard"
+"insane":                      "Difficulties/osu!/Insane"
+"normal":                      "Difficulties/osu!/Normal"
+
+"futsuu":                      "Difficulties/osu!taiko/Futsuu"
+"kantan":                      "Difficulties/osu!taiko/Kantan"
+"muzukashii":                  "Difficulties/osu!taiko/Muzukashii"
+"ura_oni":                     "Difficulties/osu!taiko/Ura_Oni"
+"inner_oni":                   "Difficulties/osu!taiko/Ura_Oni"
+
+"do_you_really_want_to_ask_peppy": "Do_you_really_want_to_ask_peppy"
+
+"faq":                         "FAQ"
+
+"game_modes":                  "Game_Modes"
+"game_mode":                   "Game_Modes"
+
+"external_ports":              "Game_Modes/External_Ports"
+"external_port":               "Game_Modes/External_Ports"
+"ports":                       "Game_Modes/External_Ports"
+"port":                        "Game_Modes/External_Ports"
+
+"opsu!":                       "Game_Modes/External_Ports/opsu!"
+"osu!droid":                   "Game_Modes/External_Ports/osu!droid"
+"osu!stream":                  "Game_Modes/External_Ports/osu!stream"
+"osu!wp":                      "Game_Modes/External_Ports/osu!wp"
+"t-aiko!":                     "Game_Modes/External_Ports/T-Aiko!"
+
+"osu!":                        "Game_Modes/osu!"
+"osu!standard":                "Game_Modes/osu!"
+
+"osu!catch":                   "Game_Modes/osu!catch"
+"osu!mania":                   "Game_Modes/osu!mania"
+"osu!taiko":                   "Game_Modes/osu!taiko"
```

Comment:
ok, I cannot add the "standard" key because it already exists for the glossary term #302